### PR TITLE
Add "Save" and "Save and Exit" Buttons for Editing

### PR DESCRIPTION
**Pull request by Daniel-Jang**
Original UAT Issue: https://github.com/cass-degrees/CASS-Degrees-Code/issues/116

I have added the 'Save' button that saves the details inserted by the user, without redirecting them back to the list page.

image

### CODE REVIEW

File: cassdegrees/templates/createcourse.html [hunk 1]
```diff
@@ -56,15 +61,21 @@
...
     </p>
 
     <script>
-        function submit_form(form_action) {
+        function submit_form(form_action, redirect) {
             if (handleRules()) {
                 document.getElementById('mainForm').action.value = form_action;
+                document.getElementById('redirect').value = redirect;
                 document.getElementById('mainForm').submit();
+                console.log(document);
```
Comment (j-selby) [inline cassdegrees/templates/createcourse.html]: Unneeded debugging (document object is going to be huge!)

**Review by j-selby — CHANGES_REQUESTED** [review]
Looking good!

**Review by JackSekoranja — APPROVED** [review]
Looks good, although I agree with James' suggested changes. I did notice that there was no 'Cancel' button when editing courses, despite there being one for Programs and Subplans. I know that isn't part of this pull request, but it's similar enough to add in this pull request since we'll need to fix it anyway.

**Comment by Daniel-Jang** [comment]
I've done what James and Jack recommended, and now is ready to merge.

**Review by j-selby — APPROVED** [review]
Looks good!

**Pull request merged**